apps/cowswap-frontend/src/modules/ordersTable/pure/OrderFillsAt/index.tsx

```diff
@@ -68,12 +69,13 @@ export function OrderFillsAt({
   onApprove,
 }: OrderFillsAtProps) {
   const { filledPercentDisplay } = order.executionData
-
+  const { t } = useLingui()
   const executionPriceInverted = isInverted ? estimatedExecutionPrice?.invert() : estimatedExecutionPrice
   const spotPriceInverted = isInverted ? spotPrice?.invert() : spotPrice
   const { feeAmount } = prices || {}
 
   const feeDifference = useFeeAmountDifference(rateInfoParams, prices)
+  const walletType = isSafeWallet ? t`Safe` : `Smart contract`
```

Comment:
In this case, then it should be the opposite, no?

Don't translate `Safe`, do translate `Smart contract`?